pageserver/src/tenant/timeline.rs

```diff
@@ -3841,6 +3867,15 @@ impl Timeline {
                                     {
                                         warn!("could not reconstruct FSM or VM key {img_key}, filling with zeros: {err:?}");
                                         ZERO_PAGE.clone()
+                                    } else if NON_INHERITED_RANGE.contains(&img_key)
```

Comment:
need fix after split missing key pull request gets merged

Reply:
Fixed in https://github.com/neondatabase/neon/pull/7404/commits/020153a376731e46d3a1b7ddd6029045d9018873